tests/helpers/run-command.js

```diff
@@ -43,18 +44,21 @@ module.exports = function run(/* command, args, options */) {
     options.log(`      Running: ${command} ${args.join(' ')} in: ${process.cwd()}`);
 
     let opts = {};
+    args = [`--unhandled-rejections=strict`, `${command}`].concat(args);
+    command = 'node';
     if (process.platform === 'win32') {
-      args = [`"${command}"`].concat(args);
-      command = 'node';
       opts.windowsVerbatimArguments = true;
       opts.stdio = [null, null, null, 'ipc'];
     }
     if (options.env) {
       opts.env = defaults(options.env, process.env);
     }
 
-    debug.info('command: %s, args: %o', command, args);
+    console.log('runCommand: %s, args: %o', command, args);
     let child = spawn(command, args, opts);
+    // ensure we tear down the child process on exit;
+    captureExit.onExit(() => killCliProcess(child));
```

Comment:
I’d prefer you to change the line above to use `execa` which automatically takes care of this bit for us

Reply:
I would prefer to not couple switching to `execa` as part of this fix. I don't mind experimenting with that in a separate/later PR.

Reply:
Ok, seems good to me. Mind making an issue at least (so between the two of us we remember to circle back).

Reply:
https://github.com/ember-cli/ember-cli/issues/9073